File: ssh_client.py
#/usr/bin/env python3

# Just so nobody can say that there is no Python file here.

import paramiko


host = '127.0.0.1' # Change this
user = 'kali' # Change this
passwd = 'kali' # Change this

client = paramiko.SSHClient()
client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
client.connect(host, username=user, password=passwd)

while True:
    stdin, stdout, stderr = client.exec_command(input('ssh> '))
    for line in stdout.readlines():
        print(line.strip())
    errors = stderr.readlines()
    if errors:
        print(errors)

File: ssh_client.pl
#!/usr/bin/env perl
# If you liked the Python SSH Client, it's here the Perl version.

use strict;
use Net::SSH::Perl;

our $host = $ARGV[0];
die "Usage: $0 <host>" unless defined $host;
print 'Username: ';
our $user = <STDIN>;
print 'Password: ';
our $passwd = <STDIN>;

our $ssh = Net::SSH::Perl->new($host);
$ssh->login($user, $passwd);

for (;;) {
	print "$host $_";
	chomp(my $cmd = <STDIN>);
	my ($stdout, $stderr) = $ssh->cmd($cmd);
	say $stdout;
	say $stderr if $stderr;
}
